cli/command/load_balancer_command_test.go

```diff
@@ -9,40 +9,64 @@ import (
 )
 
 func TestParsePort(t *testing.T) {
-	cases := map[string]*models.Port{
-		"80:80/tcp": {
-			HostPort:        80,
-			ContainerPort:   80,
-			Protocol:        "tcp",
-			CertificateName: "",
+	cases := []struct {
+		Target      string
+		Certificate string
+		Expected    models.Port
+	}{
+		{
+			Target: "80:80/tcp",
+			Expected: models.Port{
+				HostPort:      80,
+				ContainerPort: 80,
+				Protocol:      "tcp",
+			},
 		},
-		"80:80/http": {
-			HostPort:        80,
-			ContainerPort:   80,
-			Protocol:        "http",
-			CertificateName: "",
+		{
+			Target: "80:80/http",
+			Expected: models.Port{
+				HostPort:      80,
+				ContainerPort: 80,
+				Protocol:      "http",
+			},
 		},
-		"8080:80/http": {
-			HostPort:        8080,
-			ContainerPort:   80,
-			Protocol:        "http",
-			CertificateName: "",
+		{
+			Target: "8080:80/http",
+			Expected: models.Port{
+				HostPort:      8080,
+				ContainerPort: 80,
+				Protocol:      "http",
+			},
 		},
-		"443:80/https": {
-			HostPort:        443,
-			ContainerPort:   80,
-			Protocol:        "https",
-			CertificateName: "cert_name",
+		{
+			Target:      "80:80/https",
+			Certificate: "crt_name",
+			Expected: models.Port{
+				HostPort:        80,
+				ContainerPort:   80,
+				Protocol:        "https",
+				CertificateName: "crt_name",
+			},
+		},
+		{
+			Target:      "80:80/https",
+			Certificate: "arn:aws:iam::12345:server-certificate/crt_name",
```

Comment:
I know this doesn't mean much b/c it's just a string for the ARN, BUT: we should consider adding a test cases that uses acm versus iam ala `arn:aws:acm:us-west-2:236817160978:certificate/cert_name`